timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -8,6 +8,10 @@ types:
       base-type: string
       external:
         java: com.palantir.atlasdb.timelock.api.Namespace
+    GenericNamedMinTimestamp:
```

Comment:
Why not just define in conjure?

Reply:
I wanted to do validation before the object is constructed, that way I can use it in the middleware safely and only pass as string (using `name()`) in the end

If there is a way to validate things in conjure, I can do that too